en/docs/concepts/oidc/id-token.md

```diff
@@ -0,0 +1,61 @@
+# Introduction to ID Token
+
+### ID Token
+ID Token is a security token that contains claims about the authentication of an end-user by an *Authorization Server*
+which is introduced with [OpenID Connect specification](https://openid.net/specs/openid-connect-core-1_0.html#IDToken).
+This token represents as a JWT (Json Web Kit) and contains user claims to identify who the user is and it contains a unique 
+identifier that is represented in the ‘sub’ claim. The ID Token is issued from the authorization server and the audience
```

Comment:
ID Token is a security token, introduced by the [OpenID Connect specification](https://openid.net/specs/openid-connect-core-1_0.html#IDToken) that contains claims about the authentication of an end-user by an **Authorization Server.** This token is presented as a JWT (Json Web Kit) and contains user claims to identify who the user is. It also contains a unique identifier that is represented in the ‘sub’ claim.